internal/store/store.go

```diff
@@ -297,6 +306,26 @@ response:
 	return nil
 }
 
+/**
+* Creates client for v2 generated API.
+ */
+func (s *Store) createV2Client(client *http.Client) error {
+	modifiers := []apiv2.ClientModifier{
+		apiv2.UseHTTPClient(client),
+		apiv2.UseUserAgent(config.UserAgent),
+		apiv2.UseDebug(log.IsDebugLevel())}
+
+	if s.baseURL != "" {
+		modifiers = append(modifiers, apiv2.UseBaseURL(s.baseURL))
+	}
+	newClient, err := apiv2.NewClient(modifiers...)
+	if err != nil {
+		return err
+	}
+	s.clientv2 = newClient
+	return nil
```

Comment:
[nit] let's stick with the same naming conventions to avoid confusion
```suggestion
	opts := []apiv2.ClientModifier{
		apiv2.UseHTTPClient(client),
		apiv2.UseUserAgent(config.UserAgent),
		apiv2.UseDebug(log.IsDebugLevel())}

	if s.baseURL != "" {
		opts = append(modifiers, apiv2.UseBaseURL(s.baseURL))
	}
	c, err := apiv2.NewClient(opts...)
	if err != nil {
		return err
	}
	s.clientv2 = c
	return nil
```